### PR TITLE
bpo-31928: DOC: Update sys.version_info references to reflect named tu…

### PR DESCRIPTION
**Pull request by csabella**
…ple format.

https://bugs.python.org/issue31928

### CODE REVIEW

File: Doc/library/sys.rst [hunk 1]
```diff
@@ -1367,13 +1369,14 @@ always available.
...
 .. data:: version_info
 
-   A tuple containing the five components of the version number: *major*, *minor*,
-   *micro*, *releaselevel*, and *serial*.  All values except *releaselevel* are
-   integers; the release level is ``'alpha'``, ``'beta'``, ``'candidate'``, or
-   ``'final'``.  The ``version_info`` value corresponding to the Python version 2.0
-   is ``(2, 0, 0, 'final', 0)``.  The components can also be accessed by name,
-   so ``sys.version_info[0]`` is equivalent to ``sys.version_info.major``
-   and so on.
+   A :class:`~collections.namedtuple` containing the five components of the
```
Comment (serhiy-storchaka) [inline Doc/library/sys.rst:1372]: `version_info` is not a `collections.namedtuple`. It is a *named tuple*. `collections.namedtuple` implements named tuple interface, but not all named tuples are `collections.namedtuple`.

Just add the word "named" before "tuple".

File: Doc/library/sys.rst [hunk 2]
```diff
@@ -1367,13 +1369,14 @@ always available.
...
+   version number: *major*, *minor*, *micro*, *releaselevel*, and *serial*.
+   All values except *releaselevel* are integers; the release level is
+   ``'alpha'``, ``'beta'``, ``'candidate'``, or ``'final'``.  The
+   ``version_info`` value corresponding to the Python version 3.6.3 is
+   ``(major=3, minor=6, micro=3, releaselevel='final', serial=0)``.  The
```
Comment (serhiy-storchaka) [inline Doc/library/sys.rst:1377]: `(major=3, minor=6, micro=3, releaselevel='final', serial=0)` looks too verbose. And this is not a valid Python syntax. If you want to expose `repr(sys.version_info)`, you should add `sys.version_info` at the start and make it yet more verbose.

The current writing looks better to me. This is a tuple of version_info components. It is common to compare  version_info with a tuple.
```python
if sys.version_info < (3, 7):
    ...
```

File: Doc/library/sys.rst [hunk 3]
```diff
@@ -744,9 +744,11 @@ always available.
    version of the Python *language* to which the currently running
    interpreter conforms, which ``sys.version_info`` represents.  For
    example, for PyPy 1.8 ``sys.implementation.version`` might be
-   ``sys.version_info(1, 8, 0, 'final', 0)``, whereas ``sys.version_info``
-   would be ``sys.version_info(2, 7, 2, 'final', 0)``.  For CPython they
-   are the same value, since it is the reference implementation.
+   ``sys.version_info(major=1, minor=8, micro=0, releaselevel='final', serial=0)``,
```
Comment (serhiy-storchaka) [inline Doc/library/sys.rst:747]: The current writing LGTM. `sys.version_info(1, 8, 0, 'final', 0)` is not a repr, this is a constructor. `sys.version_info(major=1, minor=8, micro=0, releaselevel='final', serial=0)` looks too verbose.

Comment (csabella) [reply Doc/library/sys.rst:747]: Sorry, I thought it should look the way it is in the console when `sys.version_info` is typed.  I didn't realize it was a constructor.  I'll withdraw the PR.

**Pull request closed without merging**